e3db/tests/test_integration.py

```diff
@@ -1091,30 +1098,6 @@ def test_read_config_no_signing_keys(self):
         assert(config_client.public_signing_key == "")
         assert(config_client.private_signing_key == "")
 
-    def test_tsv1_with_note_write(self):
```

Comment:
Why are we removing this

Reply:
This was a temporary test to verify tsv1 was working without note writing. Now there is a note writing test that does the same thing. I started to update this test so it would pass and realized it was equivalent to the existing note writing test.